DangoPlop/Assets/Scripts/PlayerController.cs

```diff
@@ -75,8 +87,25 @@ public class PlayerController : MonoBehaviour {
     }
 
 	void Fire(){
+		switch (BulletType) {
+		case "DefaultFire":
+			nextFire = Time.time + FireRate;
+			Instantiate (Projectile, ProjectilePos.position, Quaternion.identity);
+			break;
+		case "DoubleShot":
+			nextFire = Time.time + DoubleShotRate;
+			Instantiate (Projectile, ProjectilePos.position, Quaternion.identity);
```

Comment:
ProjectilePos.position should be the middle point between the two shots. * | *. create a public variable for the distance from the middle to the ends and use that to calculate the spawn positions of the left and right bullet